kevm-pyk/src/kontrol/__main__.py

```diff
@@ -451,6 +452,19 @@ def parse(s: str) -> list[T]:
         action='store_true',
         help='Rekompile foundry.k even if kompiled definition already exists.',
     )
+    foundry_kompile.add_argument(
+        '--with-llvm-library',
+        dest='llvm_library',
+        default=True,
+        action='store_true',
+        help='Make kompile generate a dynamic llvm library.',
+    )
+    foundry_kompile.add_argument(
+        '--no-llvm-library',
+        dest='llvm_library',
+        action='store_false',
+        help='Do not generate a dynamic llvm library during kompile.',
+    )
```

Comment:
We're already actually building with the llvm library every time, so this isn't needed. Also, this is made more explicit here: https://github.com/runtimeverification/evm-semantics/pull/2034

So we can just remove this option altogether (and the `conflict_handler` option above).

Reply:
Thanks! Removed in https://github.com/runtimeverification/evm-semantics/pull/2044/commits/18f4e7722736dfa9f112aceb21b2d26c6e2adb19.